elpis/wrappers/input/json_to_kaldi.py

```diff
@@ -249,7 +249,7 @@ def create_kaldi_structure(input_json: str,
         only_text = []
         for file_ in all_files_in_dir:
             file_name, extension = os.path.splitext(file_)
-            print(f"file {file_name}")
+            print(f"file {file_name} {extension}")
```

Comment:
May not matter because it's a print statement, but did you want this to be e.g. `abbey_road mp4` or `abbey_road.mp4`? It's currently the former.

Reply:
Not fussed, nicer to read the code with the space :-)